mac/hid.c

```diff
@@ -226,7 +226,7 @@ static int get_string_property(IOHIDDeviceRef device, CFStringRef prop, wchar_t
 	if (!len)
 		return 0;
 
-	str = IOHIDDeviceGetProperty(device, prop);
+	str = (CFStringRef)IOHIDDeviceGetProperty(device, prop);
```

Comment:
when casting, can you keep convention `(type) variable_name;` <- please keep the space at all places

Reply:
Done